### PR TITLE
Address feedback from #40032

### PR DESCRIPTION
**Pull request by IEvangelist**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/deploying/index.md](https://github.com/dotnet/docs/blob/12a139e20b2fe6e96ba476212ac52e87c29079b6/docs/core/deploying/index.md) | [.NET application publishing overview](https://review.learn.microsoft.com/en-us/dotnet/core/deploying/index?branch=pr-en-us-40049) |
| [docs/core/docker/build-container.md](https://github.com/dotnet/docs/blob/12a139e20b2fe6e96ba476212ac52e87c29079b6/docs/core/docker/build-container.md) | [Tutorial: Containerize a .NET app](https://review.learn.microsoft.com/en-us/dotnet/core/docker/build-container?branch=pr-en-us-40049) |

### CODE REVIEW

File: docs/core/deploying/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Cross-platform binaries are created when you publish your app as [framework-depe
 
 Cross-platform binaries can be run on any operating system as long as the targeted .NET runtime is already installed. If the targeted .NET runtime isn't installed, the app may run using a newer runtime if the app is configured to roll-forward. For more information, see [framework-dependent apps roll forward](../versions/selection.md#framework-dependent-apps-roll-forward).
 
-If you're running the app in a container, you can choose to run the app directly with OS as the app host or with the `dotnet` as the host. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
+You can choose to run the app directly with a platform-specific binary as the app host or with the `dotnet` as the host. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
```
Comment (jkotas) [inline docs/core/deploying/index.md:48]:
```suggestion
You can choose to run the app as a platform-specific executable or as a cross-platform binary via `dotnet` command. There should be no app behavior difference when launching the platform-specific executable versus the `dotnet` command for ordinary server apps Launching via a platform-specific executable gives you better integration with the underlying OS. For example:
```
The terms used in the rest of this document are "platform-specific executable" and "cross-platform binary". "Host" is not used in the rest of the document. Should we be consistent?

File: docs/core/deploying/index.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Cross-platform binaries are created when you publish your app as [framework-depe
...
 
-If you're running the app in a container, you can choose to run the app directly with OS as the app host or with the `dotnet` as the host. There should be no app behavior difference when launching from via app host versus the `dotnet` host for ordinary server apps. Launching via an app host gives you better integration with the underlying OS. For example:
+You can choose to run the app as a platform-specific executable or as a cross-platform binary via `dotnet` command. There should be no app behavior difference when launching the platform-specific executable versus the `dotnet` command for ordinary server apps Launching via a platform-specific executable gives you better integration with the underlying OS. For example:
 
 - You see the app host name in your process list and not `dotnet`, which could be confusing if there's more than one.
 - You can customize the app host with OS specific features. For example, see [this discussion about configuring default stack size on Windows](https://github.com/dotnet/runtime/issues/96347#issuecomment-1981470713).
```
Comment (jkotas) [inline docs/core/deploying/index.md:51]:
```suggestion
- You can customize the platform-specific executable with OS specific features. For example, see [this discussion about configuring default stack size on Windows](https://github.com/dotnet/runtime/issues/96347#issuecomment-1981470713).
```

**Review by jkotas — APPROVED** [review]

**Comment by jkotas** [comment]
Thank you!

**Comment by IEvangelist** [comment]
> Thank you!

No way, thank you! Much appreciated.

**Pull request merged**